jinahub/crafters/ImageNormalizer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM jinaai/jina:2.0.7-py37-perf
```

Comment:
Just a general idea: We might use `master-py37-perf` as docker tag, this way we could make sure to always have the latest jina version in the Docker Containers in the hub.